setup/gcp/installer/util/utils.py

```diff
@@ -147,32 +147,43 @@ def format_resource_id(resource_type, resource_id):
     return '%s/%s' % (resource_type, resource_id)
 
 
-def format_service_acct_id(prefix, modifier, timestamp, project_id):
-    """Format the service account ids.
+def generate_service_acct_info(prefix, modifier, installation_type,
+                               timestamp, project_id):
+    """Format the service account id and name.
+
+    Id will contain the timestamp due to the uniqueness requirement
+    and name will contain installation type for clarity purpose.
 
     Args:
-        prefix (str): The prefix of the account id
-        modifier (str): Access level of the account
-        timestamp (str): Timestamp of the class
-        project_id (str): Id of the project on GCP
+        prefix (str): The prefix of the account id and account name.
+        modifier (str): Access level of the account.
+        installation_type (str): Type of the installation (client/server).
+        timestamp (str): The timestamp.
+        project_id (str): Id of the project on GCP.
 
     Returns:
-        str: Service account id
+        str: Service account id.
+        str: Service account name.
     """
 
-    return full_service_acct_email(
+    account_id = full_service_acct_email(
```

Comment:
for this kind of methods, please start with verb: e.g. `create_full_service_acct_email()`